### PR TITLE
Travis

### PR DESCRIPTION
**Pull request by dgalbraith33**
Added Continuous Integration and fixed formatting issues

### CODE REVIEW

File: src/main/java/codeu/model/store/persistence/PersistentDataStore.java [hunk 1]
```diff
@@ -51,8 +49,8 @@ public PersistentDataStore() {
   /**
    * Loads all User objects from the Datastore service and returns them in a List.
    *
-   * @throws PersistentDataStoreException if an error was detected during the load from the
-   *     Datastore service
+   * @throws codeu.model.store.persistence.PersistentDataStoreException if an error was detected
```
Comment (Rafer45) [inline src/main/java/codeu/model/store/persistence/PersistentDataStore.java]: This looks a bit weird -- why not just import PersistentDataStoreException?

File: src/test/java/codeu/controller/AdminServletTest.java [hunk 2]
```diff
@@ -54,9 +53,9 @@ public void setup() {
     Mockito.when(mockRequest.getSession()).thenReturn(mockSession);
 
     mockRequestDispatcher = Mockito.mock(RequestDispatcher.class);
-     /* When AdminServlet tries to forward to the dispatcher of "/WEB-INF/view/admin.jsp", just
-      * return the mock dispatcher.
-      */
+    /* When AdminServlet tries to forward to the dispatcher of "/WEB-INF/view/admin.jsp", just
```
Comment (Rafer45) [inline src/test/java/codeu/controller/AdminServletTest.java:115]: Are comments supposed to be
```java
/* like
 * this?
 */
```
or
```java
/**
 * like
 * this?
 */
```

**Review by Rafer45 — DISMISSED** [review]

File: src/test/java/codeu/controller/AdminServletTest.java (hunk 2 above)
Comment (dgalbraith33) [reply src/test/java/codeu/controller/AdminServletTest.java:115]: Not sure it matters for inline comments. The latter is for javadocs I believe

**Review by Rafer45 — APPROVED** [review]
LGTM!

**Pull request merged**